ml-agents/mlagents/trainers/policy/torch_policy.py

```diff
@@ -95,31 +95,29 @@ def export_memory_size(self) -> int:
 
     def _split_decision_step(
         self, decision_requests: DecisionSteps
-    ) -> Tuple[SplitObservations, np.ndarray]:
-        vec_vis_obs = SplitObservations.from_observations(decision_requests.obs)
+    ) -> Tuple[List[np.array], np.ndarray]:
+        obs = decision_requests.obs
         mask = None
         if not self.use_continuous_act:
             mask = torch.ones([len(decision_requests), np.sum(self.act_size)])
             if decision_requests.action_mask is not None:
                 mask = torch.as_tensor(
                     1 - np.concatenate(decision_requests.action_mask, axis=1)
                 )
-        return vec_vis_obs, mask
+        return obs, mask
 
-    def update_normalization(self, vector_obs: np.ndarray) -> None:
+    def update_normalization(self, buffer: AgentBuffer) -> None:
         """
         If this policy normalizes vector observations, this will update the norm values in the graph.
         :param vector_obs: The vector observations to add to the running estimate of the distribution.
         """
-        vector_obs = [torch.as_tensor(vector_obs)]
         if self.use_vec_obs and self.normalize:
-            self.actor_critic.update_normalization(vector_obs)
+            self.actor_critic.update_normalization(buffer)
```

Comment:
Not sure how I feel about passing the whole buffer (rather than just the obs) to the normalizer. But probably doesn't really matter

Reply:
The way the observations as list is made, there can be multiple vector observations per Agent, this allows to normalize potentially all of the observations that can be normalized.

Reply:
I'm thinking going forward we can replace this with a data structure (e.g. ObservationsTuple) that would be created from the buffer, and can pass in here. But this is OK for this PR